README.md

```diff
@@ -212,49 +188,96 @@ inputs = generate_random_inputs(onnx_filepath, batch_size)
 engine = compile_model(onnx_filepath, batch_size)
 outputs = engine.run(inputs)
 ```
-**Compatibility/Support Notes**
+Compatibility/Support Notes
 - ONNX version 1.5-1.7
 - ONNX opset version 11+
 - ONNX IR version has not been tested at this time
 
-For a more in-depth read on available APIs and workflows, check out the [examples](https://github.com/neuralmagic/deepsparse/blob/main/examples/) and [DeepSparse Engine documentation.](https://docs.neuralmagic.com/deepsparse)
+The [GitHub repository](https://github.com/neuralmagic/deepsparse) includes package APIs along with examples to quickly get started benchmarking and inferencing sparse models.
+
+__ __
+
+## Scheduling Single-Stream, Multi-Stream, and Elastic Inference
+
+The DeepSparse Engine offers up to three types of inferences based on your use case. You can read more details here: [Inference Types](https://github.com/neuralmagic/deepsparse/blob/main/docs/source/scheduler.md).
+
+1 ⚡ Single-stream scheduling: the latency/synchronous scenario, requests execute serially. [`default`]
+
+<img src="https://raw.githubusercontent.com/neuralmagic/deepsparse/main/docs/source/single-stream.png" alt="single stream diagram" />
+
+Use Case: It's highly optimized for minimum per-request latency, using all of the system's resources provided to it on every request it gets.
+
+2 ⚡ Multi-stream scheduling: the throughput/asynchronous scenario, requests execute in parallel.
+
+<img src="https://raw.githubusercontent.com/neuralmagic/deepsparse/main/docs/source/multi-stream.png" alt="multi stream diagram" />
+
+PRO TIP: The most common use cases for the multi-stream scheduler are where parallelism is low with respect to core count, and where requests need to be made asynchronously without time to batch them.
+
+3 ⚡ Elastic scheduling: requests execute in parallel, but not multiplexed on individual NUMA nodes.
+
+Use Case: A workload that might benefit from the elastic scheduler is one in which multiple requests need to be handled simultaneously, but where performance is hindered when those requests have to share an L3 cache.
+__ __
+
+## CPU Hardware Support
```

Comment:
Do we care that the hardware citation is so far down on this page? What motivates a reader to follow all the content upfront only to conclude the reader may not even have the right hardware?

Reply:
CPU hardware support link to docs is top of the page.